cmd/login.go

```diff
@@ -93,9 +93,6 @@ func NewLoginCmd(cmd *cobra.Command, args []string) (*LoginCmd, error) {
 	}
 
 	password := viper.GetString(argLoginPassword)
-	if password == "" {
-		return nil, errors.New("No password set")
-	}
```

Comment:
looks good to me. on master, the same if with the `username` is not present as well.

Reply:
it's different because on master we also prompt for username, but in 1.5.x we don't

Reply:
@kjaskiewiczz could you cherry this instead: https://github.com/mendersoftware/mender-cli/commit/a03979a283e04af4cbbebfb81a2e197f41cca02c ?

Reply:
@oleorhagen  That can be cherry-picked (or we can reopen your PR) but requires a `Changelog` tag